### DISCUSSION TITLE
PG::ConnectionBad:

### DISCUSSION THREAD

**Discussion by soffoalbert** [ubicloud/ubicloud#1380] [Q&A]
Hi there

Please help

I am getting the following error when starting the docker services on my Hetzner rented service. 

Sequel::DatabaseConnectionError: PG::ConnectionBad: connection to server at "", port 5432 failed: FATAL:  password authentication failed for user "clover" (Sequel::DatabaseConnectionError)
/usr/local/bundle/gems/pg-1.5.6/lib/pg/connection.rb:701:in `async_connect_or_reset'
/usr/local/bundle/gems/pg-1.5.6/lib/pg/connection.rb:833:in `connect_to_hosts'
/usr/local/bundle/gems/pg-1.5.6/lib/pg/connection.rb:764:in `new'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/adapters/postgres.rb:231:in `connect'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/connecting.rb:248:in `new_connection'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/connection_pool.rb:163:in `make_new'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/connection_pool/threaded.rb:209:in `assign_connection'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/connection_pool/threaded.rb:139:in `acquire'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/connection_pool/threaded.rb:91:in `hold'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/connecting.rb:293:in `synchronize'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/connecting.rb:302:in `test_connection'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/misc.rb:189:in `initialize'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/connecting.rb:57:in `new'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/connecting.rb:57:in `connect'
/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/core.rb:124:in `connect'
/app/db.rb:10:in `<top (required)>'
/app/Rakefile:11:in `require_relative'
/app/Rakefile:11:in `block in <top (required)>'
/app/Rakefile:76:in `block in <top (required)>'
/usr/local/bundle/gems/rake-13.1.0/exe/rake:27:in `<top (required)>'

Thanks in advance
Georges

**Comment by enescakir** [comment]
Hi @soffoalbert, It appears the console can't connect to the database due to an authentication failure. This issue often arises when `demo/generate_env` is run multiple times, causing the password in `demo/.env` and the docker database container to mismatch. This typically results from leftover docker volumes. Could you try deleting the docker volume and running the demo again?

`docker volume rm demo_postgres`
